.github/pull_request_template.md

```diff
@@ -21,6 +21,7 @@ Before you mark the PR ready for review, please make sure that:
   - If the change does not require a CHANGELOG.md entry, do one of the following:
     - Add `[skip changelog]` to the PR title
     - Add the label `skip/changelog` to the PR
+- [ ] Add the label `need/very-expensive-tests` if the PR requires running very expensive tests
```

Comment:
```suggestion
- [ ] Run [very expensive tests](https://github.com/search?q=repo%3Afilecoin-project%2Flotus+VeryExpensive&type=code) if useful or skip
   - If touching codepaths affecting "very expensive tests", add the label `need/very-expensive-tests`.
   - Otherwise skip (do nothing)
```

To make it clear what tests we're talking about.
Frame this such that people always check the box depending on which path they go (adding label or not).  (That seems like how some of the other items are phrased like CHANGELOG updates.